src/PokeList.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 14/11/2018
+ * Time: 11:48
+ */
+
+namespace Pokedex;
+
+class PokeList {
+
+    private $pokemon;
```

Comment:
A bit too general, not clear that this refers to a list of pokemon.

Reply:
Solved in c032493

Reply:
Solved in c032493